### DISCUSSION TITLE
Translate in other languages than English?

### DISCUSSION THREAD

**Discussion by pragomer** [openai/whisper#2394] [Q&A]
Hi, quick question:
Whisper can translate "any" speech to English.
Can it also translate "any language" in other languages than English? For example German?

Thanks already for your help

P.

**Comment by glangford** [comment]
See these past discussions.

* https://github.com/openai/whisper/discussions/649
* https://github.com/openai/whisper/discussions/49
* https://github.com/openai/whisper/discussions/1160
* https://github.com/openai/whisper/discussions/665

and note in particular
* https://github.com/openai/whisper/discussions/649#discussioncomment-4325122
